### PR TITLE
Od/pcc token commands

### PR DESCRIPTION
**Pull request by Aumkar**

### CODE REVIEW

File: pcc/src/cli/commands/login.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+import open from 'open';
+import destroyer from 'server-destroy';
+import { readFileSync } from 'fs';
+import nunjucks from 'nunjucks';
+nunjucks.configure({ autoescape: true });
+
+const OAUTH_SCOPES = [
+  'https://www.googleapis.com/auth/drive.file',
```
Comment (kevinstubbs) [inline pcc/src/cli/commands/login.ts:13]: Why do we need this scope?

File: pcc/src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import os from 'os';
+import path from 'path';
+
+export const PCC_ROOT_DIR = path.join(os.homedir(), '.pcc');
+export const GOOGLE_CLIENT_ID =
+  '142470191541-8o14j77pvagisc66s48kl4ub91f9c7b8.apps.googleusercontent.com';
+export const GOOGLE_REDIRECT_URI = 'http://localhost:3030/oauth-redirect';
```
Comment (kevinstubbs) [inline pcc/src/constants.ts:7]: I thought that Google stopped allowing localhost redirect urls?
https://developers.google.com/identity/protocols/oauth2/resources/loopback-migration

Extra reasoning
https://stackoverflow.com/questions/67343399/why-is-it-acceptable-to-use-the-http-scheme-for-oauth-loopback-interface-redir

File: pcc/src/lib/addonApiHelper.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import axios, { HttpStatusCode } from 'axios';
+import { getLocalAuthDetails } from './localStorage';
+import { HTTPNotFound } from '../cli/exceptions';
+
+const API_KEY_ENDPOINT =
+  'https://us-central1-pantheon-content-cloud-staging.cloudfunctions.net/addOnApi/api-key';
+const OAUTH_ENDPOINT =
+  'https://us-central1-pantheon-content-cloud-staging.cloudfunctions.net/addOnApi/oauth';
```
Comment (kevinstubbs) [inline pcc/src/lib/addonApiHelper.ts:8]: 1.) These need to point to prod by default.
2.) There needs to be a way for us to change to staging for development purposes.

File: pcc/src/cli/commands/login.ts (hunk 1 above)
Comment (Aumkar) [reply pcc/src/cli/commands/login.ts:13]: I just included all scopes which are required for Add On except Add on specific ones.
But yeah, this is not used atm.

File: pcc/src/lib/addonApiHelper.ts (hunk 3 above)
Comment (Aumkar) [reply pcc/src/lib/addonApiHelper.ts:8]: Oh yes, I missed this. Good catch.

**Review by kevinstubbs — APPROVED** [review]

**Pull request merged**